docs/DeveloperGuide.md

```diff
@@ -241,14 +245,14 @@ Step 5: The set of `UserImage` is then converted into an `ArrayList` and the fir
 ## Statistics feature/Window
 <img src="images/user-guide/Stats.png" height="400px">
 
-The `stats` opens a new `StatisticsWindow` that displays a pie chart with the data of the number of sellers & buyers in the 5 different regions, namely {North, South, East, West, Central}.
+The `stats` opens a new Statistics window that displays a pie chart with the data of the number of sellers & buyers in the 5 different regions, namely {North, South, East, West, Central}.
```

Comment:
Maybe adding the word command to become "The `stats` command opens" will be better?